functions.php

```diff
@@ -724,11 +725,15 @@ function export_graphs(&$export, $export_path) {
 				}
 			}
 
+            export_debug('There are ' . sizeof($graphs) . ' graphs not in hosts to export for all trees');
```

Comment:
Can you change the spaces to tabs to keep with the formatting standards?  There's even a line where you've not changed the code but seem to have introduced spaces before any text for some reason. 👍

Reply:
vim settings.  Correcting.

Reply:
Corrected.